### ISSUE TITLE
How to remove the operation of app initialization to obtain notification permission

### ISSUE THREAD

**Issue by caijiami** [wonday/react-native-aliyun-push#154]
Android app markets don't allow users to get access right away, and privacy policies in China are now very strict

**Comment by caijiami** [comment]
@wonday @JadeZhu @haipengz @chenweigh Thank you bosses

**Comment by BridgeEast** [comment]
define RCT_EXPORT_METHOD to do the initialization, call it when you need. @caijiami

**Comment by caijiami** [comment]
> define RCT_EXPORT_METHOD to do the initialization, call it when you need. @caijiami

Hello, can you give me specific steps? I am not very familiar with native code

**Comment by BridgeEast** [comment]
> > define RCT_EXPORT_METHOD to do the initialization, call it when you need. @caijiami
> 
> Hello, can you give me specific steps? I am not very familiar with native code
Native Modules Intro https://reactnative.dev/docs/native-modules-intro
just like the ios example
```objective_c
RCT_EXPORT_METHOD(initJPush) {
  dispatch_async(dispatch_get_main_queue(), ^{

  //  // APNS
    JPUSHRegisterEntity * entity = [[JPUSHRegisterEntity alloc] init];
    if (@available(iOS 12.0, *)) {
      entity.types = JPAuthorizationOptionAlert|JPAuthorizationOptionBadge|JPAuthorizationOptionSound|JPAuthorizationOptionProvidesAppNotificationSettings;
    }
    [JPUSHService registerForRemoteNotificationConfig:entity delegate:self];
  });
}
```
then 
use `NativeModules.JPushHelperModule.initJPush();` in js module
